pkg/migrations/mls/20240528181851_init-schema.up.sql

```diff
@@ -0,0 +1,71 @@
+SET statement_timeout = 0;
+
+--bun:split
+CREATE TABLE installations(
+	id BYTEA PRIMARY KEY,
+	created_at BIGINT NOT NULL,
+	updated_at BIGINT NOT NULL,
+	inbox_id BYTEA NOT NULL,
+	key_package BYTEA NOT NULL,
+	expiration BIGINT NOT NULL
+);
+
+--bun:split
+CREATE TABLE group_messages(
+	id BIGSERIAL PRIMARY KEY,
+	created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+	group_id BYTEA NOT NULL,
+	data BYTEA NOT NULL,
+	group_id_data_hash BYTEA NOT NULL
+);
+
+--bun:split
+CREATE INDEX idx_group_messages_group_id_id ON group_messages(group_id, id);
```

Comment:
This index is different from before. We were previously indexing on `(group_id, created_at)` and it never got updated when we changed the sort field on our queries.